base/applications/winhlp32/CMakeLists.txt

```diff
@@ -11,16 +11,24 @@ list(APPEND SOURCE
     callback.c
     hlpfile.c
     macro.c
-    winhelp.c
-    precomp.h)
+    winhelp.c)
+
+list(APPEND PCH_SKIP_SOURCE
+    macro.lex.yy.c
+    string.c)
 
 # macro.lex.yy.c has been generated with relative file paths...
 set_source_files_properties(macro.lex.yy.c PROPERTIES COMPILE_FLAGS "-UREACTOS_SOURCE_DIR -DREACTOS_SOURCE_DIR=\"\\\".\\\"\"")
 
 add_rc_deps(rsrc.rc ${CMAKE_CURRENT_SOURCE_DIR}/res/winhelp.ico)
-add_executable(winhlp32 ${SOURCE} macro.lex.yy.c string.c rsrc.rc)
+
+add_executable(winhlp32
+    ${SOURCE}
+    ${PCH_SKIP_SOURCE}
+    rsrc.rc)
```

Comment:
CMake's PCH doesn't apply force-includes to .rc files, does it?

Reply:
Yeap, I checked before letting them in :)